PocketKit/Sources/SharedPocketKit/User.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import SwiftUI
+
+public enum Status: String {
+    case premium = "premium"
+    case free = "free"
+    case unknown = "unknown"
+}
+
+public protocol UserProtocol {
```

Comment:
Basically, and just more readable (to me). It kind of feels like the equivalent of doing something like 

`class ViewModelClass { }`

which just doesn't seem right. 

We know it's a protocol because it's defined as such, so there's no need to add the suffix. That's my thinking, at least.

Reply:
makes sense, thanks! will do this going forward